app/models/async_info.rb

```diff
@@ -49,6 +49,7 @@ def to_hash
       feed_style: feed_style_preference,
       created_at: user.created_at,
       admin: user.any_admin?,
+      hidden_tags: user.cached_antifollowed_tag_names.presence,
```

Comment:
Just a thought, I'm wondering if could have used the existing `followed_tags` and filter by points < 0 instead of passing through the hidden_tags attribute?

Reply:
@Ridhwana Ah, that definitely could have worked, had I thought of it. I don't think the duplication hurts us, but do you think it'd be worth refactoring to use the existing `followed_tags` ?